Ast/src/Parser.cpp

```diff
@@ -2197,11 +2207,25 @@ AstExpr* Parser::parseSimpleExpr()
     {
         return parseString();
     }
+    else if (FFlag::LuauInterpolatedStringBaseSupport && lexer.current().type == Lexeme::InterpStringBegin)
+    {
+        return parseInterpString();
+    }
     else if (lexer.current().type == Lexeme::BrokenString)
     {
         nextLexeme();
         return reportExprError(start, {}, "Malformed string");
     }
+    else if (lexer.current().type == Lexeme::BrokenInterpDoubleBrace)
+    {
+        nextLexeme();
+        return reportExprError(start, {}, ERROR_INVALID_INTERP_DOUBLE_BRACE);
+    }
+    else if (lexer.current().type == Lexeme::BrokenInterpNoFormat)
+    {
+        nextLexeme();
+        return reportExprError(start, {}, "Interpolated strings must contain expressions, and cannot be constant");
```

Comment:
Is "cannot be constant" redundant here?

Reply:
A bit, I wrote it with sympathy to younger developers who might not understand what "must contain expressions" means, I'm unsure if that's a bit too compiler enthusiast on its own.

Reply:
For the record, we're going to allow this as per #615 if that gets ratified on its own.